oktaawscli/okta_auth.py

```diff
@@ -143,18 +145,18 @@ def verify_single_factor(self, factor, state_token):
             if resp_json['status'] == "SUCCESS":
                 return resp_json['sessionToken']
             elif resp_json['status'] == "MFA_CHALLENGE" and factor['factorType'] !='u2f':
-                print("Waiting for push verification...")
+                self.logger.info("Waiting for push verification...")
```

Comment:
I'm on the fence about this one. I feel like this is a normal operational message that the user should see, but they'd need to have the `--verbose` flag set in order to. Without it, they'll likely wonder what's happening, unless they happen to see the push notification on their device.